.gitignore

```diff
@@ -1,9 +1,146 @@
-### IntelliJ ###
-.idea/
```

Comment:
@kwk would you mind leaving the `.idea/` for the goland IDE in the `.gitignore`
I see you adding other IDE which is fine but let's leave that one

Reply:
Why do we want to put any IDE setting under version control? I don't see why we should do that?

Reply:
No
Like you ignore eclipse, emac, ignore also goland idea

Reply:
Ah :) sorry, I misread the comment as: "please add it". On it.

Reply:
I've updated the gitignore.io URL to include intellij: https://www.gitignore.io/?templates=go,vim,git,macos,linux,emacs,windows,eclipse,intellij+all,visualstudiocode Are you fine with this @corinnekrych ?